static/js/src/entry/circle/TopForm.vue

```diff
@@ -161,15 +161,15 @@
 </template>
 
 <script>
-import Hidden from '../../elements/Hidden.vue';
-import LocalHidden from '../../elements/LocalHidden.vue';
-import PayFees from '../../elements/PayFees.vue';
-import TextInput from '../../elements/TextInput.vue';
-import ManualPay from '../../elements/ManualPay.vue';
-import ManualSubmit from '../../elements/ManualSubmit.vue';
-import NativePay from '../../elements/NativePay.vue';
+import Hidden from '../../connected-elements/Hidden.vue';
+import LocalHidden from '../../local-elements/Hidden.vue';
```

Comment:
here, are we saying connected means hooked up to some API? And local is just for our app?

Reply:
Connected means connected to the Vuex store. Local means relying on local component state. I'll document that.

Reply:
Noted!